lib/rucio/db/sqla/migrate_repo/versions/b818052fa670_add_index_to_quarantined_replicas.py

```diff
@@ -0,0 +1,37 @@
+'''
+ Copyright European Organization for Nuclear Research (CERN)
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ You may not use this file except in compliance with the License.
+ You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
lib/rucio/db/sqla/migrate_repo/script.py.mako should be modified to reflect the right header and the header should be changed here as well.